cpu/cc2538/periph/uart.c

```diff
@@ -316,3 +222,37 @@ void uart_poweroff(uart_t uart)
 {
```

Comment:
you should also implement the power_on and power_off functions while you at it. Probably they can be just on-liners as in 
````c
assert(uart < UART_NUMOF);
uart_config[uart].dev->cc2538_uart_ctl.CTLbits.UARTEN = [0|1];
```

Reply:
Also, bitmasks rather than each bit on one line

Reply:
ok, I've done that. Also the clock on/off. However, they're untested...

Reply:
test them :-)

One thing that just came to mind: it might be useful to make sure that the TX FIFO is empty before putting the device into poweroff mode, right?

Reply:
#8447 makes testing this hopefully easier for you...